### PR TITLE
Allow RGB Class Config to be None

### PR DESCRIPTION
**Pull request by jamesmcclain**
## Overview

Brief description of what this PR does, and why it is needed.

It seems as though `self.rgb_class_config` is formally allowed to be `None`, but the call to `ensure_null_class` assumes that it is non-null.

### Checklist

- [x] Added `needs-backport` label if PR is bug fix that applies to previous minor release
- [x] Ran scripts/format_code and committed any changes
- [x] Documentation updated if needed
- [x] PR has a name that won't get you publicly shamed for vagueness

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

### CODE REVIEW

**Review by AdeelH — APPROVED** [review]

**Comment by AdeelH** [comment]
This PR fixes an oversight from [this commit](https://github.com/azavea/raster-vision/pull/1260/commits/1dfff1d36004568ffff010bc0cb1db08eb92ca6f
) from #1260 and is thus part of the fix for #1259.

**Comment by jamesmcclain** [comment]
Thanks for the quick review, I'll wait until after #1323 goes in before merging.

**Comment by AdeelH** [comment]
@jamesmcclain, please merge this PR.

**Comment by jamesmcclain** [comment]
@AdeelH Will merge after CI

**Pull request merged**
